src/pal/inc/pal.h

```diff
@@ -5038,9 +5048,15 @@ PAL_SetTerminationRequestHandler(
 class CatchHardwareExceptionHolder
 {
 public:
-    CatchHardwareExceptionHolder();
+    CatchHardwareExceptionHolder()
+    {
+        PAL_CatchHardwareExceptionHolderEnter();
```

Comment:
Does moving these implementations into the header relate to the renaming or is this something else? (not implying you need to move it, just trying to understand)

Reply:
We were exporting the C++ decorated names for the constructor/destructor and I didn't want to depend on the compiler's name decoration format anymore.